core/vibe/core/drivers/libevent2.d

```diff
@@ -62,12 +62,18 @@ version(Windows)
 version(OSX)
 {
 	import std.c.osx.socket : IP_ADD_MEMBERSHIP, IP_MULTICAST_LOOP;
+	static const IP_TOS = 3;
+} else version(Linux)
+{
+    import std.c.linux.socket : IP_ADD_MEMBERSHIP, IP_MULTICAST_LOOP;
```

Comment:
I came to the same conclusions. I have a unfinished PR laying around adding those and more to druntime. I just need to add all the different OSes.